deployment/ccip/changeset/testhelpers/test_helpers.go

```diff
@@ -482,6 +482,87 @@ func AddLane(
 	require.NoError(t, err)
 }
 
+// UpdateLane updates the lane to enable or disable between the source and destination chains.
+// It applies a series of changesets to update the router ramps, fee quoter, on-ramp destinations, and off-ramp sources.
+//
+// Parameters:
+//   - t: The testing object.
+//   - e: The deployed environment.
+//   - src: The source chain selector.
+//   - dest: The destination chain selector.
+//   - isTestRouter: Boolean flag indicating if the test router is used.
+//   - isEnabled: Boolean flag indicating if the lane is enabled.
+func UpdateLane(
+	t *testing.T,
+	e *DeployedEnv,
+	src, dest uint64,
+	isTestRouter, isEnabled bool,
+) {
+	var err error
+	defaultFeeQuoterDestConfig := changeset.DefaultFeeQuoterDestChainConfig()
+	defaultFeeQuoterDestConfig.IsEnabled = isEnabled
+	apps := []commoncs.ChangesetApplication{
+		{
+			Changeset: commoncs.WrapChangeSet(changeset.UpdateRouterRampsChangeset),
+			Config: changeset.UpdateRouterRampsConfig{
+				UpdatesByChain: map[uint64]changeset.RouterUpdates{
+					// onRamp update on source chain
+					src: {
+						OnRampUpdates: map[uint64]bool{
+							dest: isEnabled,
+						},
+					},
+					// offramp update on dest chain
+					src: {
+						OffRampUpdates: map[uint64]bool{
+							dest: isEnabled,
+						},
```

Comment:
your comment does not match the selector key , your updating `src`.
Also this part is not required. We don't want to disable offRamp on dest router as the inflight requests and all manual exec requests will be stuck if offRamp is removed from dest router.